app/models/maestrano/connector/rails/synchronization.rb

```diff
@@ -41,7 +41,7 @@ def set_partial
     def clean_synchronizations
       count = self.organization.synchronizations.count
       if count > 100
-        self.organization.synchronizations.limit(count - 100).destroy_all
+        self.organization.synchronizations.order('id ASC').limit(count - 100).destroy_all
```

Comment:
Keep only 100 synchronizations per organization, so deleting the n - 100 sync when creating a new one (it's an after_create)